src/nupic/types/Sdr.hpp

```diff
@@ -492,10 +496,10 @@ class SparseDistributedRepresentation : public Serializable
      * Save (serialize) the current state of the SDR to the specified file.
      * This method can NOT save callbacks!  Only the dimensions and current data
      * are saved.
-     * 
+     *
      * @param stream A valid output stream, such as an open file.
      */
-    void save(std::ostream &outStream) const override;
+    void save(std::ostream &outStream) const override;   // will be removed later
```

Comment:
You should delete the old save & load methods as you go.  Then implement save/load in the serialization class by calling the new cereal methods.  That way the new methods get tested.